packages/trader/src/Modules/Trading/Components/Form/ContractType/ContractTypeInfo/contract-type-info-header.tsx

```diff
@@ -1,20 +1,48 @@
 import React from 'react';
 import { Icon, Text } from '@deriv/components';
+import { clickAndKeyEventHandler } from '@deriv/shared';
 
 type THeader = {
     onClickGoBack?: () => void;
+    onClickCross?: () => void;
+    should_render_arrow?: boolean;
+    should_render_cross?: boolean;
     text_size?: string;
     title: string;
 };
 
-const Header = ({ onClickGoBack, text_size = 's', title }: THeader) => (
+const Header = ({
+    onClickGoBack,
+    onClickCross,
+    should_render_arrow = true,
+    should_render_cross = false,
+    text_size = 's',
+    title,
+}: THeader) => (
     <div className='contract-type-info__action-bar'>
-        <span className='contract-type-info__icon' id='dt_contract_info_back_nav' onClick={onClickGoBack}>
-            <Icon icon='IcArrowLeftBold' />
-        </span>
+        {should_render_arrow && (
+            <span
+                className='contract-type-info__icon'
+                id='dt_contract_info_back_nav'
```

Comment:
```suggestion
                data-testid='dt_contract_info_back_nav'
```
did you intend to add a data-testid? :)

Reply:
No, id was in original file. Will delete as it's unused